src/redprl/atomic_judgment.sig

```diff
@@ -41,12 +41,12 @@ struct
       *)
      | SUB_TYPE of (abt * abt)
 
-     (* `SUB_UNIVERSE (a, k)`
-      *   `a` is a sub-universe of U^`k`.
+     (* `SUB_KIND (a, k)`
+      *   `a` is a sub-universe of the universe of `k` types at the omega level.
```

Comment:
@jonsterling Please check that `SUB_KIND` is the proper name for this semantics.

Reply:
I think it mostly makes sense. The only thing that is a bit weird is I would expect subkind to have two kinds as its arguments. Would it make more sense to say `HAS_KIND`? That is, the universe `a` can be regarded as having the kind `k`.

Reply:
Wouldn't that be misunderstood as the universe itself being of kind `k`?

Reply:
@favonia Haha, you are right 😉